github/apps.go

```diff
@@ -36,12 +36,35 @@ type InstallationToken struct {
 	ExpiresAt *time.Time `json:"expires_at,omitempty"`
 }
 
-// InstallationPermissions lists the permissions for metadata, contents, issues and single file for an installation.
+// InstallationPermissions lists the repository and organization permissions for an installation.
+//
+// Permission names taken from:
+//   https://developer.github.com/v3/apps/permissions/
+//   https://developer.github.com/enterprise/v3/apps/permissions/
 type InstallationPermissions struct {
-	Metadata   *string `json:"metadata,omitempty"`
-	Contents   *string `json:"contents,omitempty"`
-	Issues     *string `json:"issues,omitempty"`
-	SingleFile *string `json:"single_file,omitempty"`
+	Administration              *string `json:"administration,omitempty"`
+	Checks                      *string `json:"checks,omitempty"`
+	Contents                    *string `json:"contents,omitempty"`
+	ContentReferences           *string `json:"content_references,omitempty"`
+	Deployments                 *string `json:"deployments,omitempty"`
+	Issues                      *string `json:"issues,omitempty"`
+	Metadata                    *string `json:"metadata,omitempty"`
```

Comment:
According to the following documentation:
https://developer.github.com/apps/building-github-apps/creating-github-apps-using-url-parameters/#github-app-permissions

It seems like this should be `RepositoryMetadata`, and more importantly, the JSON key should be `repository_metadata`?

Reply:
@itsdalmo that seems to be an error in the docs, the key from the response is definitely `metadata`.

Reply:
Thanks, @itsdalmo and @tarebyte!

Would one of you please volunteer to contact support@github.com and let them know about the documentation error?  They are typically very responsive, friendly, and helpful, and this would be appreciated feedback.   Thank you!

Reply:
> Would one of you please volunteer to contact support@github.com and let them know about the documentation error?

I can file an internal issue, I work for GitHub 😃 

> They are typically very responsive, friendly, and helpful, and this would be appreciated feedback. Thank you!

💖

Reply:
Oh, uh, Wow!  I'm humbled and honored that you are helping us out here, @tarebyte!!!

You and your team have been an absolute joy to work with!  I have a bunch of virtual friends at GitHub Tech Support.  Thanks for all you do!

So let's see... all I need is another LGTM and we are good to merge.

Reply:
> Oh, uh, Wow! I'm humbled and honored that you are helping us out here, @tarebyte!!!

> You and your team have been an absolute joy to work with! I have a bunch of virtual friends at GitHub Tech Support. Thanks for all you do!

It's my pleasure! Thank you for working on such a ✨ library